### PR TITLE
changed single polygon to merged polygon POI if uploaded multi polygon during project area edit

### PR DESCRIPTION
**Pull request by Sujanadh**

### CODE REVIEW

**Review by nrjadkry — APPROVED** [review]

File: src/backend/app/projects/project_crud.py [hunk 1]
```diff
@@ -1322,153 +1356,153 @@ def generate_appuser_files(
         - form_type: weather the form is xls, xlsx or xml
         - background_task_id: the task_id of the background task running this function.
     """
-    try:
```
Comment (nrjadkry) [inline src/backend/app/projects/project_crud.py:1325]: @Sujanadh You have commented the try except block

**Review by nrjadkry — CHANGES_REQUESTED** [review]

**Review by nrjadkry — APPROVED** [review]

**Comment by nrjadkry** [comment]
@spwoodcock Have you looked into this PR. Looks good to me.

**Review by spwoodcock — APPROVED** [review]

**Comment by spwoodcock** [comment]
Thanks for the prompt Niraj! Hectic few weeks 😄

**Comment by spwoodcock** [comment]
And of course, thank you @Sujanadh for the PR!

**Pull request merged**